homeassistant/components/totalconnect/strings.json

```
@@ -76,5 +77,10 @@
         "name": "Bypass"
       }
     }
+  },
+  "exceptions": {
+    "invalid_pin": {
+      "message": "User entered incorrect alarm PIN code."
```

```suggestion
      "message": "Incorrect code entered"
```
Or something which is not "third-person" typed